gamestonk_terminal/portfolio/README.md

```diff
@@ -0,0 +1,40 @@
+#PORTFOLIO
+
+This was developed to interface with Robinhood.  
+
+Once this screen is accessed, the first command to be run is 
+````
+login
+````
+Which will use your robinhood login information, which should be stored as environment variables or in the config_terminal.py
+
+Currently there are 2 options:
+* [hold](#hold)
+    * Look at current (stock only) holdings
+* [rhhist](#rhhist)
+    * Get and plot rhhistorical portfolio
+
+## hold <a name="hold"></a>
+
+Displays current holdings to the console:
+
+````
+usage: hold
+````
+There are no additional flags.  This just prints all current stonks, their last price, previous close, your equity and
+the % change from previous close.
+
+Example:
+![hold](https://user-images.githubusercontent.com/18151143/111685384-3c6ab080-87fe-11eb-80ce-9b256c396bf2.png)
```

Comment:
I usually change VSCode theme to light in order to have a white background. Not important though.

Reply:
Its a known fact that Dark Mode increases profits by 5% minimum :)

Reply:
In order for my Portfolio to get green I better use Vantablack (https://www.businessinsider.com/the-surprising-origin-of-the-darkest-color-on-earth-2019-10?r=US&IR=T) :D

Reply:
Also I feel like a fraud for not having any GME.  paper hands here.